docker/prod/setup/postinstall.sh

```diff
@@ -27,7 +27,7 @@ if test -f ./docker/prod/setup/postinstall-$PLATFORM.sh ; then
 fi
 
 echo " ---> Precompiling assets. This will take a while..."
-./docker/prod/setup/postinstall-common.sh > /tmp/dockerize.log || display_error
+./docker/prod/setup/postinstall-common.sh
 
 rm -f /tmp/dockerize.log
```

Comment:
@b12f If the logs are no longer written into a log file you don't need to remove the non-existent file either!

Reply:
oh yeah!